### DISCUSSION TITLE
Error Code 39 Meaning?

### DISCUSSION THREAD

**Discussion by vasatrya** [cosmos/community#52] [Validators, Nodes, Relayers, etc...]
Hi, I am trying to unjail a node on a blockchain created via Cosmos, and am getting the following error:

“failed to execute message; message index: 0: cannot delegate to validators with invalid (zero) ex-rate”

What does this error code exactly mean?

**Comment by julienrbrt** [comment]
How many delegations has the validator?

https://github.com/cosmos/cosmos-sdk/blob/f0d6e7717351ed998575e9cb2beafaa626ab78f2/x/staking/keeper/delegation.go#L681:L686

**Reply by vasatrya** [reply]
How can I check?

The chain I am running the node on requires 50k tokens.  I recently moved the 50k tokens - if I update the staking address would that make a difference?
